test/e2e/cluster.sh

```diff
@@ -125,6 +126,108 @@ function rotateCertificates {
+    -e IS_JENKINS="${IS_JENKINS}" \
+    -e TEST_PVC="${TEST_PVC}" \
+    -e SKIP_TEST="true" \
+    -e GINKGO_FAIL_FAST="${GINKGO_FAIL_FAST}" \
+    -e GINKGO_FOCUS="${GINKGO_FOCUS}" \
+    -e GINKGO_SKIP="${GINKGO_SKIP}" \
+    -e API_PROFILE="${API_PROFILE}" \
+    -e CUSTOM_CLOUD_NAME="${ENVIRONMENT_NAME}" \
+    -e IDENTITY_SYSTEM="${IDENTITY_SYSTEM}" \
+    -e AUTHENTICATION_METHOD="${AUTHENTICATION_METHOD}" \
+    -e LOCATION="${LOCATION}" \
+    -e CUSTOM_CLOUD_CLIENT_ID="${CUSTOM_CLOUD_CLIENT_ID}" \
+    -e CUSTOM_CLOUD_SECRET="${CUSTOM_CLOUD_SECRET}" \
+    -e PORTAL_ENDPOINT="${PORTAL_ENDPOINT}" \
+    -e SERVICE_MANAGEMENT_ENDPOINT="${SERVICE_MANAGEMENT_ENDPOINT}" \
+    -e RESOURCE_MANAGER_ENDPOINT="${RESOURCE_MANAGER_ENDPOINT}" \
+    -e STORAGE_ENDPOINT_SUFFIX="${STORAGE_ENDPOINT_SUFFIX}" \
+    -e KEY_VAULT_DNS_SUFFIX="${KEY_VAULT_DNS_SUFFIX}" \
+    -e ACTIVE_DIRECTORY_ENDPOINT="${ACTIVE_DIRECTORY_ENDPOINT}" \
+    -e GALLERY_ENDPOINT="${GALLERY_ENDPOINT}" \
+    -e GRAPH_ENDPOINT="${GRAPH_ENDPOINT}" \
+    -e SERVICE_MANAGEMENT_VM_DNS_SUFFIX="${SERVICE_MANAGEMENT_VM_DNS_SUFFIX}" \
+    -e RESOURCE_MANAGER_VM_DNS_SUFFIX="${RESOURCE_MANAGER_VM_DNS_SUFFIX}" \
+    -e STABILITY_ITERATIONS=${STABILITY_ITERATIONS} \
+    -e STABILITY_ITERATIONS_SUCCESS_RATE=${STABILITY_ITERATIONS_SUCCESS_RATE} \
+    -e SINGLE_COMMAND_TIMEOUT_MINUTES=${SINGLE_COMMAND_TIMEOUT_MINUTES} \
+    -e STABILITY_TIMEOUT_SECONDS=${STABILITY_TIMEOUT_SECONDS} \
+    -e ARC_CLIENT_ID=${ARC_CLIENT_ID:-$AZURE_CLIENT_ID} \
+    -e ARC_CLIENT_SECRET=${ARC_CLIENT_SECRET:-$AZURE_CLIENT_SECRET} \
+    -e ARC_SUBSCRIPTION_ID=${ARC_SUBSCRIPTION_ID:-$AZURE_SUBSCRIPTION_ID} \
+    -e ARC_LOCATION=${ARC_LOCATION:-$LOCATION} \
+    -e ARC_TENANT_ID=${ARC_TENANT_ID:-$AZURE_TENANT_ID} \
+    -e LINUX_CONTAINERD_URL=${LINUX_CONTAINERD_URL} \
+    -e WINDOWS_CONTAINERD_URL=${WINDOWS_CONTAINERD_URL} \
+    -e LINUX_MOBY_URL=${LINUX_MOBY_URL} \
+    -e LINUX_RUNC_URL=${LINUX_RUNC_URL} \
+    -e VALIDATE_CPU_LOAD=${VALIDATE_CPU_LOAD} \
+    -e RUN_VMSS_NODE_PROTOTYPE=${RUN_VMSS_NODE_PROTOTYPE} \
+    -e KAMINO_VMSS_PROTOTYPE_LOCAL_CHART_PATH=${KAMINO_VMSS_PROTOTYPE_LOCAL_CHART_PATH} \
+    -e KAMINO_VMSS_PROTOTYPE_IMAGE_REGISTRY=${KAMINO_VMSS_PROTOTYPE_IMAGE_REGISTRY} \
+    -e KAMINO_VMSS_PROTOTYPE_IMAGE_REPOSITORY=${KAMINO_VMSS_PROTOTYPE_IMAGE_REPOSITORY} \
+    -e KAMINO_VMSS_PROTOTYPE_IMAGE_TAG=${KAMINO_VMSS_PROTOTYPE_IMAGE_TAG} \
+    -e CUSTOM_KUBE_PROXY_IMAGE=${CUSTOM_KUBE_PROXY_IMAGE} \
+    -e CUSTOM_KUBE_APISERVER_IMAGE=${CUSTOM_KUBE_APISERVER_IMAGE} \
+    -e CUSTOM_KUBE_SCHEDULER_IMAGE=${CUSTOM_KUBE_SCHEDULER_IMAGE} \
+    -e CUSTOM_KUBE_CONTROLLER_MANAGER_IMAGE=${CUSTOM_KUBE_CONTROLLER_MANAGER_IMAGE} \
+    -e CUSTOM_KUBE_BINARY_URL=${CUSTOM_KUBE_BINARY_URL} \
+    -e CUSTOM_WINDOWS_PACKAGE_URL=${CUSTOM_WINDOWS_PACKAGE_URL} \
+    -e AZURE_CORE_ONLY_SHOW_ERRORS="True" \
+    "${DEV_IMAGE}" make test-kubernetes || exit 1
+
+  echo "Installing azuredisk-csi-driver..."
```

Comment:
I think this is unnecessary as you can leverage `cluster-init` in this case: https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md#components